report_qweb_pdf_watermark/models/report.py

```diff
@@ -8,6 +8,12 @@
 from PIL import Image
 from StringIO import StringIO
 from odoo import api, models, tools
+from PIL import PdfImagePlugin # flake8: noqa
+
```

Comment:
Comment added on the code.
Codecov will fail because previous commits were failing.